src/game/g_target.cpp

```diff
@@ -1618,6 +1618,26 @@ void SP_target_remove_portals(gentity_t *self) {
   }
 }
 
+void target_portal_relay_use(gentity_t *self, gentity_t *other,
+                               gentity_t *activator) {
+  if (!activator ||
+      !activator->client ||
+      activator->client->sess.sessionTeam == TEAM_SPECTATOR ||
+      !self->target) {
+    return;
+  }
+  // don't need to check health, see comment in player_die
```

Comment:
I have no idea what this comment means even after reading the comment in `player_die`, what health? Dead players don't activate entities anyway so my mind instantly goes to `self->health` but that doesn't make any sense.

Reply:
Various delayed activations like a button or target_delay could be activated with `numPortals = 0` if you die (ungibbed) before the activation. I can clarify this in the comment, but if there's a chance that you will be revived, despite losing your portals I think the portal count shouldn't be reset.

Reply:
e.g. Shoot 2 portals, press button, hit a hurt-brush, and when the event activates your numportals would be 0.

Reply:
reseted portals in player_die anyways